### PR TITLE
automatically setup and update RHUI repos

### PR DESCRIPTION
**Pull request by mcalmer**
# Description

RHUI setup will get some automation using the PAYG client connection feature

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default) (not ported yet)
- [x] Manager-4.3  (not before 4.3.7)
- [ ] Manager-4.2

# Links

Development Task: https://github.com/SUSE/spacewalk/issues/21247
4.3 development PR: https://github.com/SUSE/spacewalk/pull/20669
master developement PR: TBD

### CODE REVIEW

**Comment by mcalmer** [comment]
RedHat ship the client certificates via repositories.
To keep this working, the customer must not only keep the instance running, but also keep it up-to-date.

We should add a note about this. Not sure how exact this note needs to be?
Every Cloud has its own names and repositories where these packages are delivered.

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 1]
```diff
@@ -61,127 +53,92 @@ Clients using Satellite certificates are not supported with {productname} Server
...
+. Navigate to the page section [guimenu]``Bastion SSH connection data``.
+. In the [guimenu]``Host`` field, enter the bastion hostname.
+. In the [guimenu]``SSH Port`` field, enter the bastion port number.
+. In the [guimenu]``User`` field, enter the bastion username.
+. In the [guimenu]``Password`` field, enter the bastion password.
+. In the [guimenu]``SSH Private Key`` field, enter the bastion key.
+. In the [guimenu]``SSH Private Key Passphrase`` field, enter the bastion key passphrase.
 
-To manage repositories on your {redhat} client, you need to import the CA and entitlement certificates to the {productname} Server.
+Complete the setup process with with <<proc-finishing-rhui-connecting>>.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:115]:
```suggestion
Complete the setup with with <<proc-finishing-rhui-connecting>>.
```

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 2]
```diff
@@ -61,127 +53,92 @@ Clients using Satellite certificates are not supported with {productname} Server
...
 
-This requires that you perform the import procedure three times, to create three entries, one of each for the entitlement certificate, the entitlement key, and the {redhat} certificate.
+[[proc-finishing-rhui-connecting]]
+.Procedure: Finishing {redhat} connecting
+[role=procedure]
+. To complete adding new {redhat} connection data, click btn:[Create].
+. Return to {payg} connection data [guimenu]``Details`` page.
+    The updated connection status is displayed on the top section named [guimenu]``Information``.
+. Connection status is shown in [guimenu]``Admin > Setup Wizard > Pay-as-you-go`` screen too.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:123]:
```suggestion
. Connection status is shown in [guimenu]``Admin > Setup Wizard > Pay-as-you-go`` screen, too.
```

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 3]
```diff
@@ -61,127 +53,92 @@ Clients using Satellite certificates are not supported with {productname} Server
...
+. If the authentication data for the instance are correct, the column [guimenu]``Status`` shows "Credentials successfully updated."
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:124]:
```suggestion
. If the authentication data for the instance is correct, the column [guimenu]``Status`` shows [literal]``Credentials successfully updated.``
```

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 4]
```diff
@@ -61,127 +53,92 @@ Clients using Satellite certificates are not supported with {productname} Server
...
 
+[IMPORTANT]
+====
+If the invalid data are entered at any point, the newly created instance is shown in [guimenu]``Admin > Setup Wizard > {payg}``, with column [guimenu]``Status`` displaying error message.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:128]:
```suggestion
If invalid data is entered at any point, the newly created instance is shown in [guimenu]``Admin > Setup Wizard > {payg}``, with column [guimenu]``Status`` displaying an error message.
```

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 5]
```diff
@@ -61,127 +53,92 @@ Clients using Satellite certificates are not supported with {productname} Server
...
-* In the [guimenu]``Type`` field, select [systemitem]``SSL``.
-* In the [guimenu]``Select file to upload`` field, browse to the location where you saved the {redhat} certificate, and select the certificate file.
-. Click btn:[Create Key].
+[IMPORTANT]
+====
+A {redhat} connection will create custom repositories which are owned by organization 1 by default.
+If a different organization should own the autogenerated repositories, configure [systemitem]``java.rhui_default_org_id`` in ``/etc/rhn/rhn.conf``
+====
 
+This only defines and updates the repositories. If you want to use a repository for a managed client, you need to specify a Software Channel and connect the repositories to it.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:141]:
```suggestion
This only defines and updates the repositories.
If you want to use a repository for a managed client, you need to specify a Software Channel and connect the repositories to it.
```

**Review by keichwa — APPROVED** [review]
Some cosmetic wording suggestion.

**Review by jcayouette — APPROVED** [review]

**Review by keichwa — APPROVED** [review]
Thanks!  imo, it is ready to be merged.  Either you or Joseph please can press the squash+merge button.  [I'm out today.]

**Pull request merged**